.github/workflows/go.yml

```diff
@@ -0,0 +1,76 @@
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: Build All
+      run: |
+        GOOS=darwin GOARCH=386 go build -o sort.darwin-386 .
+        GOOS=darwin GOARCH=amd64 go build -o sort.darwin-amd64 .
+        GOOS=linux GOARCH=386 go build -o sort.linux-386 .
+        GOOS=linux GOARCH=amd64 go build -o sort.linux-amd64 .
+        GOOS=linux GOARCH=arm go build -o sort.linux-arm .
+        GOOS=linux GOARCH=arm64 go build -o sort.linux-arm64 .
+        GOOS=linux GOARCH=ppc64 go build -o sort.linux-ppc64 .
+        GOOS=linux GOARCH=ppc64le go build -o sort.linux-ppc64le .
+        GOOS=linux GOARCH=mips go build -o sort.linux-mips .
+        GOOS=linux GOARCH=mipsle go build -o sort.linux-mipsle .
+        GOOS=linux GOARCH=mips64 go build -o sort.linux-mips64 .
+        GOOS=linux GOARCH=mips64le go build -o sort.linux-mips64le .
+        GOOS=linux GOARCH=s390x go build -o sort.linux-s390x .
+        GOOS=aix GOARCH=ppc64 go build -o sort.aix-ppc64 .
+        GOOS=openbsd GOARCH=386 go build -o sort.openbsd-386 .
+        GOOS=openbsd GOARCH=amd64 go build -o sort.openbsd-amd64 .
+        GOOS=openbsd GOARCH=arm go build -o sort.openbsd-arm .
+        GOOS=openbsd GOARCH=arm64 go build -o sort.openbsd-arm64 .
+        GOOS=netbsd GOARCH=386 go build -o sort.netbsd-386 .
+        GOOS=netbsd GOARCH=amd64 go build -o sort.netbsd-amd64 .
+        GOOS=netbsd GOARCH=arm go build -o sort.netbsd-arm .
+        GOOS=windows GOARCH=386 go build -o sort.windows-386 .
+        GOOS=windows GOARCH=386 go build -o sort.plan9-amd64 .
+        GOOS=plan9 GOARCH=386 go build -o sort.plan9-386 .
+        GOOS=plan9 GOARCH=amd64 go build -o sort.plan9-amd64 .
+        GOOS=plan9 GOARCH=arm go build -o sort.plan9-arm .
+        GOOS=solaris GOARCH=amd64 go build -o sort.solaris-amd64 .
+        GOOS=js GOARCH=wasm go build -o sort.js-wasm .
+        GOOS=freebsd GOARCH=386 go build -o sort.freebsd-386 .
+        GOOS=freebsd GOARCH=amd64 go build -o sort.freebsd-amd64 .
+        GOOS=freebsd GOARCH=arm go build -o sort.freebsd-arm .
+        GOOS=illumos GOARCH=amd64 go build -o sort.illumos-amd64 .
+        GOOS=dragonfly GOARCH=amd64 go build -o sort.dragonfly-amd64 .
+        
+        #GOOS=android GOARCH=386 go build -o sort.android-386 .
+        #GOOS=android GOARCH=amd64 go build -o sort.android-amd64 .
+        #GOOS=android GOARCH=arm go build -o sort.android-arm . 
+        #GOOS=android GOARCH=arm64 go build -o sort.android-arm64 .
+        #GOOS=darwin GOARCH=arm go build -o sort.darwin-arm .
+        #GOOS=darwin GOARCH=arm64 go build -o sort.darwin-arm64 .
+
+ 
```

Comment:
* [ ] Drop the empty lines
* [ ] Order all not by not-commented-out/commented-out, but like in the table you posted in the issue

Reply:
Drop the empty lines.